packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -14,7 +14,8 @@ export const SelectedItemDefault: (props: IPeoplePickerItemProps) => JSX.Element
     item,
     onRemoveItem,
     index,
-    selected
+    selected,
+    removeButtonAriaLabel = ''
```

Comment:
I just wanted to ensure nothing would break; I'll take out the default value and verify that it's all good regardless.